api/maven-api-core/src/main/java/org/apache/maven/api/JavaPathType.java

```diff
@@ -187,31 +247,38 @@ public Optional<String> option() {
      *
      * @param paths the path to format as a tool option
      * @return the option associated to this path type followed by the given path elements,
-     *         or an empty string if there is no path element
+     *         or an empty array if there is no path element
      * @throws IllegalStateException if no option is associated to this path type
      */
     @Nonnull
     @Override
-    public String option(Iterable<? extends Path> paths) {
+    public String[] option(Iterable<? extends Path> paths) {
```

Comment:
I wonder if this method should be moved out of the API.

Reply:
The presence or not of this method depends on the [comment below](https://github.com/apache/maven/pull/1501#discussion_r1590432486).